.vscode/launch.json

```diff
@@ -55,11 +55,29 @@
             "console": "internalConsole",
             "stopAtEntry": false
         },
+        {
+            "name": "Launch Ruby",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            "program": "${workspaceFolder}/src/kiota/bin/Debug/net5.0/kiota.dll",
+            "args": ["--openapi",
+                 "${workspaceFolder}/../msgraph-sdk-powershell/openApiDocs/v1.0/mail.yml",
+                 "--language",
+                 "ruby",
+                 "-o",
+                 "${workspaceFolder}/samples/msgraph-mail/ruby",
+                 "-n",
+                 "graphrubyv4.utilities" ],
```

Comment:
it'd be interesting to test with the ruby packaging format as well as ruby dev will probably pass in that format (::)